### DISCUSSION TITLE
Github Copilot Extension is not found?

### DISCUSSION THREAD

**Discussion by jjangga0214** [VSCodium/vscodium#961] [Q&A] [answered]
I wanted to install Github Copilot, but could not find the extension.
On vscode, I could do so.

OS: macOS 12.1 Monterey (Apple Silicon ARM M1)

Does this happen because of lack of `vscode.github-authentication`, which, unlike vscode, vscodium can not ship?

Thanks

**About Vscodium:**

Version: 1.63.2
Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3
Date: 2021-12-17T00:22:15.020Z (2 days ago)
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.2.0

**Comment by ghost** [comment, marked as answer]
It doesn't use Visual Studio Code Marketplace.

**Reply by GraysonMakesGames** [reply]
Yes, this doesn't use the VS Marketplace (VS Code extensions are uploaded to the VS Marketplace) and I don't think Copilot was uploaded to what provider Codium uses, I forgot which one)
